### PR TITLE
add tips

### PR DESCRIPTION
**Pull request by dabuside**
add the ``--section='common'`` argument infomation
fix issue https://github.com/jupyter/notebook/issues/3517#issuecomment-380205926

### CODE REVIEW

File: docs/source/extending/frontend_extensions.rst [hunk 1]
```diff
@@ -219,11 +219,11 @@ rather than copying it, so there's no need to reinstall after changes.
 To use your extension, you'll also need to **enable** it, which tells the
 notebook interface to load it. You can do that with another command::
 
-    jupyter nbextension enable my_extension/main [--sys-prefix]
+    jupyter nbextension enable my_extension/main [--sys-prefix|--section='common']
```
Comment (takluyver) [inline docs/source/extending/frontend_extensions.rst]: It's not mutually exclusive from the `--sys-prefix` option, so the normal way is to put each one in its own `[square brackets]`.

File: docs/source/extending/frontend_extensions.rst [hunk 2]
```diff
@@ -219,11 +219,11 @@ rather than copying it, so there's no need to reinstall after changes.
...
 
 The argument refers to the Javascript module containing your
 ``load_ipython_extension`` function, which is ``my_extension/main.js`` in this
-example. There is a corresponding ``disable`` command to stop using an
+example. The ``--section='common'`` argument will affect all pages. There is a corresponding ``disable`` command to stop using an
```
Comment (takluyver) [inline docs/source/extending/frontend_extensions.rst]: I think this should mention that the default is for the extension only to be loaded on the notebook view - that is probably what most extensions want.

File: docs/source/extending/frontend_extensions.rst [hunk 3]
```diff
@@ -219,11 +219,11 @@ rather than copying it, so there's no need to reinstall after changes.
 To use your extension, you'll also need to **enable** it, which tells the
 notebook interface to load it. You can do that with another command::
 
-    jupyter nbextension enable my_extension/main [--sys-prefix|--section='common']
+    jupyter nbextension enable my_extension/main [--sys-prefix][--section='common']
 
 The argument refers to the Javascript module containing your
 ``load_ipython_extension`` function, which is ``my_extension/main.js`` in this
-example. The ``--section='common'`` argument will affect all pages. There is a corresponding ``disable`` command to stop using an
+example. The ``--section='common'`` argument will affect all pages, by default it will be loaded on the notebook view only. There is a corresponding ``disable`` command to stop using an
```
Comment (takluyver) [inline docs/source/extending/frontend_extensions.rst]: Can you wrap this so the lines are max 80 columns? Thanks.

**Comment by dabuside** [comment]
have any idea to make extension only loaded in the dashboard(file browser) page? [--section='dashboard']arguments will affect notebook view as well

**Comment by takluyver** [comment]
`tree` should be the name (like in the URL). The list of accepted values is here:

https://github.com/jupyter/notebook/blob/f69ce9fcadd46c282450ff57bf4d855b833f9cae/notebook/nbextensions.py#L37

**Pull request merged**